spec/jobs/google_scraping_job_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe GoogleScrapingJob, type: :job do
+  include ActiveJob::TestHelper
+
+  describe '#perform' do
+    it 'enqueues google scraping job' do
+      user = Fabricate(:user)
+      keyword = Fabricate(:keyword, user_id: user[:id], keyword: 'AWS')
+
+      expect do
+        GoogleScrapingJob.perform_later(keyword.id, keyword.keyword)
+      end.to have_enqueued_job(GoogleScrapingJob)
+
+      assert_enqueued_with(job: GoogleScrapingJob, args: [keyword.id, 'AWS'])
+    end
+
+    it 'creates GoogleScrapingService instance' do
+      user = Fabricate(:user)
+      keyword = Fabricate(:keyword, user_id: user[:id], keyword: 'AWS')
+
+      VCR.use_cassette('with_top_position_adwords', record: :none) do
+        GoogleScrapingJob.perform_now(keyword.id, keyword.keyword)
+      end
+
+      result = Keyword.find(keyword.id)
+
+      expect(result.status).to eql('completed')
+    end
+
+    context 'given error raising' do
+      it 'retrys the job 5 times' do
```

Comment:
This feature is Rails's feature, we can trust Rail on that, so IMO we can go without this test 😀

Reply:
Fixed in c5ee583574df5e37583605efbb12a85b868dee0b